### PR TITLE
Additional fixups

### PR DESCRIPTION
**Pull request by jmh530**

### CODE REVIEW

File: source/mir/stat/constant.d [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import mir.math.constant: PI;
+
+///
+enum real LOGPI = log(PI);
+///
+enum real LOGSQRT2PI = 0.91893853320467274178032973640561764L; // log(sqrt(2pi))
+///
+enum real SQRTPI = sqrt(PI);
+///
+enum real SQRTPIINV = 1 / SQRTPI;
```
Comment (9il) [inline source/mir/stat/constant.d]: Worth to use hexadecimal constants from Wolfram, more precise.

Comment (jmh530) [reply source/mir/stat/constant.d]: I can do things like "log(pi) to hex" in Wolfram alpha, but those aren't the same format as the ones you use in `mir.math.constant`. Is there something I'm missing?

Comment (9il) [reply source/mir/stat/constant.d]: For log(pi) you can add `0x` prefix and `p0` suffix.

 `0x1.250d048e7a1bd0bd5f956c6a843f4p0`

Comment (jmh530) [reply source/mir/stat/constant.d]: Thanks. Fixed in https://github.com/libmir/mir-stat/pull/77/commits/e738a84fe9b08eb10315bfb7e37a2b32370c36ec

**Comment by jmh530** [comment]
@9il Do you know how I would make this macro
`MATHREF = $(GREF_ALTTEXT mir-algorithm, $(TT $2), $2, mir, math, $1)$(NBSP)`
also bold the text?

**Comment by jmh530** [comment]
The bolding is fixed in the latest version of the docs

**Pull request merged**